R/row-redux.R

```diff
@@ -156,3 +156,12 @@ col_concat <- function(data, sep=""){
   apply(data, 1, paste0, collapse=sep)
 }
 attr(col_concat, "call") <- "col_concat"
+
+#' @export
+is_col_duplicated <- function(data, allow.na = FALSE) {
```

Comment:
This name does not reflect what the function does  - it returns a list that for each for each row shows if any value is duplicated column wise. Kind of difficult to express. We need to think about either changing the name or the logic.

Reply:
My suggestions for the function name:
`any_value_has_duplicates_in_col`
`any_value_has_duplicates_in_own_col`

Or change a logic to function:
`values_are_unique_in_cols`
`values_are_unique_in_own_cols`
and use negation of it.

Reply:
@krystian8207 first option looks good:
```
any_value_has_duplicates_in_col
any_value_has_duplicates_in_own_col
```